ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -287,12 +284,12 @@ export class RegistrationDashboard extends React.Component<RegistrationDashboard
       }
       {!betaAccessGranted &&
         <div data-test-id='beta-access-warning'
-             style={{...baseStyles.card, ...styles.warningModal}}>
+             style={{...baseStyles.card, ...styles.warningModal, margin: '1rem 0 0'}}>
```

Comment:
Could you just make this `marginTop`? Or do you explicitly want to override the others with 0? (Same for above)